doc/guides/connection_management.rst

```diff
@@ -1,26 +1,28 @@
+.. _connection_mgnt:
+
 Connection Management
 =====================
 
 From NEST 2.4 onwards the old connection routines (i.e.
-``(Random)ConvergentConnect``, ``(Random)DivergentConnect`` and plain
-``Connect``) are replaced by one unified ``Connect`` function. In
+`(Random)ConvergentConnect`, `(Random)DivergentConnect` and plain
+`Connect`) are replaced by one unified `Connect` function. In
 `SLI <an-introduction-to-sli.md>`__ ,the old syntax of the function
 still works, while in `PyNEST <introduction-to-pynest.md>`__, the
-``Connect()`` function has been renamed to ``OneToOneConnect()``.
+`Connect()` function has been renamed to `OneToOneConnect()`.
```

Comment:
for existing functions, it would be nice to use the proper form to link directly to the corresponding docstring
either through ``:func:`nest.Connect`` or ``:func:`nest.lib.hl_api_connections.Connect`` depending on whether or not the full path is necessary.
do this to all functions in the API if possible

Reply:
For creating these links, I'd like to wait for another PR -- we have a script that can search through all function occurrences and replace with the proper format.